org/qmckl_sherman_morrison_woodbury.org

```diff
@@ -817,18 +1001,20 @@ qmckl_exit_code qmckl_sherman_morrison_smw32s_c(const qmckl_context context,
     }
   }
 
-  if (remainder == 2) { // Apply last remaining block of 2 updates with Woodbury 2x2 kernel
+  // Apply last remaining block of 2 updates with Woodbury 2x2 kernel
+  if (remainder == 2) {
     const double *Updates_2block = &Updates[n_of_3blocks * length_3block];
     const uint64_t *Updates_index_2block = &Updates_index[3 * n_of_3blocks];
-    rc = qmckl_woodbury_2_c(context, Dim, Updates_2block, Updates_index_2block, breakdown, Slater_inv);
+    rc = qmckl_woodbury_2_c_(context, &Dim, Updates_2block, Updates_index_2block, &breakdown, Slater_inv);
     if (rc != 0) { // Send the entire block to slagel_splitting
       uint64_t l = 0;
       (void) qmckl_slagel_splitting_c(Dim, 2, Updates_2block, Updates_index_2block,
               breakdown, Slater_inv, later_updates + (Dim * later), later_index + later, &l);
       later = later + l;
     }
   }
-  else if (remainder == 1) { // Apply last remaining update with slagel_splitting
+  // Apply last remaining update with slagel_splitting
+  else if (remainder == 1) {
     const double *Updates_1block = &Updates[n_of_3blocks * length_3block];
```

Comment:
Yes, I like this style too. I find it easier to read comments on a separate line.

Reply:
Your welcome ;)